### PR TITLE
Add task 3

### PR DESCRIPTION
**Pull request by puptup**

### CODE REVIEW

File: GoHomeWork3/main.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package main
+
+import "fmt"
+
+type Point struct {
+	x, y int
+}
+
+type Square struct {
```
Comment (ikamornikau) [inline GoHomeWork3/main.go:9]: Could you please add a comment to describe the type?

File: GoHomeWork3/main.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	start Point
+	a     uint
```
Comment (ikamornikau) [inline GoHomeWork3/main.go:11]: What is `a`? Could you add more clarity by adding a comment?

File: GoHomeWork3/main.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+func (p Square) Perimeter() uint {
+	return p.a * 4
+}
+
+func (p Square) End() Point {
```
Comment (ikamornikau) [inline GoHomeWork3/main.go:18]: What is `End()`? Could you please add a comment to clarify it?

File: GoHomeWork3/main.go (hunk 1 above)
Comment (puptup) [reply GoHomeWork3/main.go:12]: i fixed it all :)

**Pull request merged**